integration-tests/src/contract.rs

```diff
@@ -0,0 +1,56 @@
+use cosmwasm_std::{Addr, Coin};
+use cw_multi_test::{App, AppResponse, Executor};
+use error_stack::{report, Result};
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
+pub trait Contract {
+    type QMsg;
+    type ExMsg;
+    type Err;
```

Comment:
can you check if all contracts use axelar_wasm_std::ContractError as output? In that case we can bake that type directly into the trait instead of making it an associate type

Reply:
yes, all of them use `use axelar_wasm_std_derive::IntoContractError`.

Reply:
replace `Self::Err` with `axelar_wasm_std::ContractError` and remove `type Err`

Reply:
ok.